### PR TITLE
Added automation test for submitting Transitional Accommodation CAS3 referral and verification in Delius

### PR DESCRIPTION
**Pull request by smalepati99**
Added an automation test script to facilitate the submission of a Transitional Accommodation CAS3 referral and subsequent verification in Delius.

This automation test code covers the following steps:

Step 1: Log in to Delius
Step 2: Create a person/offender in Delius
Step 3: Create a custodial event in Delius
Step 4: Create and Book prisoner
Step 5: Log in to CAS3 and submit a CAS3 referral
Step 6: Log back in to Delius to verify contacts

The test script has been structured and commented to enhance readability and maintainability. Please review and provide feedback.

### CODE REVIEW

File: steps/cas3-transitional-accommodation/application.ts [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+            "Test summary of person's index offence(s) and offending history any history of arson, violent offences, weapons etc"
+        )
+    await page.getByRole('button', { name: 'Save and continue' }).click()
+    await expect(page).toHaveTitle(
+        'Which of the following best describes the sentence type? - Transitional Accommodation (CAS3)'
+    )
+    await page.getByRole('radio', { name: 'Standard determinate custody', exact: true }).check()
+    await page.getByRole('button', { name: 'Save and continue' }).click()
+    await expect(page).toHaveTitle('What is the sentence length? - Transitional Accommodation (CAS3)')
+    await page.getByLabel('Years').fill(String(0))
```
Comment (marcus-bcl) [inline steps/cas3-transitional-accommodation/application.ts]: Do you need to wrap this in String() ?  Could you just pass `'0'` ?

File: steps/cas3-transitional-accommodation/login.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { expect, type Page } from '@playwright/test'
+
+export const login = async (page: Page) => {
+    await page.goto(process.env.CAS3_SHORT_TERM_ACCOMMODATION_URL)
```
Comment (marcus-bcl) [inline steps/cas3-transitional-accommodation/login.ts]: Should this be called `CAS3_TRANSITIONAL_ACCOMMODATION_URL`?

File: steps/cas3-transitional-accommodation/application.ts (hunk 1 above)
Comment (marcus-bcl) [reply steps/cas3-transitional-accommodation/application.ts]: Similarly throughout this file

**Review by marcus-bcl — COMMENTED** [review]
Test looks good - just a couple of minor nitpicks.

File: steps/cas3-transitional-accommodation/login.ts (hunk 2 above)
Comment (smalepati99) [reply steps/cas3-transitional-accommodation/login.ts]: Updated and pushed

File: steps/cas3-transitional-accommodation/application.ts (hunk 1 above)
Comment (smalepati99) [reply steps/cas3-transitional-accommodation/application.ts]: I will update it now

Comment (smalepati99) [reply steps/cas3-transitional-accommodation/application.ts]: Updated all the file and pushed now

**Review by marcus-bcl — APPROVED** [review]

**Pull request merged**